compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -54,7 +54,7 @@ static LogicalResult setRootConfig(func::FuncOp entryPointFn,
         entryPointFn, matmulOp, tileSizes,
         IREE::Codegen::DispatchLoweringPassPipeline::None);
   } else if (usePassPipeline == AIEPassPipeline::PackPipeline) {
-    SmallVector<int64_t> TileSizeLevel0 = {16, 64};
+    SmallVector<int64_t> TileSizeLevel0 = {16, 256};
```

Comment:
Please try to add a separate flag to switch between single core and multicore

Reply:
You're right, I will work out a solution for that.

Reply:
To merge this PR, you can change it back to 64 and add another PR for multicore case.

Reply:
Oh, I see you also change the test. Then that should be fine. But it's better we can add both single and multicore cases and both tests.

Reply:
Just pushed the changes, from my side this PR is ready to land!